CHANGELOG.md

```diff
@@ -1,17 +1,18 @@
 # Changelog
 
-### 1.7.0 - Release Candidate 1
+### 1.7.0
 
 OVERVIEW:
 
-Release 1.7.0-rc1 includes:
+Release 1.7.0 includes:
 * Support for TCP, UDP, and TLS Passthrough load balancing with the new configuration resources: TransportServer and GlobalConfiguration. The resources allow users to deliver complex, non-HTTP-based applications from Kubernetes using the NGINX Ingress Controller.
 * Support for error pages in VirtualServer and VirtualServerRoute resources. A user can now specify custom error responses for errors returned by backend applications or generated by NGINX, such as a 502 response.
 * Improved validation of VirtualServer and VirtualServerRoute resources. kubectl and the Kubernetes API server can now detect violations of the structure of VirtualServer/VirtualServerRoute resources and return an error.
+* Support for an OpenShift operator which manages the lifecycle of the Ingress Controller on OpenShift. See the [NGINX Ingress Operator GitHub repo](https://github.com/nginxinc/nginx-ingress-operator).
```

Comment:
The operator works for k8s too, should we mention it here just in case this causes confusion? (people might think the operator only works in Openshift)

Reply:
good suggestion!